website-custom-views/src/content/api-reference/custom-view-config.mdx

```diff
@@ -549,6 +549,15 @@ The following list contains all the currently available locators:
 * `customers.customer_addresses.new_adress`
 * `customers.customer_addresses.edit_address`
 * `customers.customer_orders.order_detail`
+* `customers.customer_details.business_units`
```

Comment:
Just to keep consistency, this one should be placed after the `customers.customer_details.orders` one.

Reply:
Thanks @CarlosCortizasCT, I fixed this in [c22e686](https://github.com/commercetools/merchant-center-application-kit/pull/3377/commits/c22e686fe149ed2f608997f237444616cfb7137e).